core/domain/suggestion_jobs_one_off.py

```diff
@@ -471,3 +475,74 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
+
+
+class DeleteAllQuestionAndInvalidSuggestionsOneOffJob(
+        jobs.BaseMapReduceOneOffJobManager):
+    """One-off job to delete all questions and invalid suggestion models in the
+    server.
+
+    This job deletes following suggestion and connected models:
+        - Deletes all the question suggestions as the old question suggestions
```

Comment:
Are we going to reconstitute this data? How?

Reply:
Currently, the question suggestion project is not released to the end-user so we don't have data in the prod server but we do have some data in the test server. The question suggestion data in the test server have state_schema_version: 1 which is not a valid number. We cannot (easily) find the valid state schema number for these models as the state schema version is changed/incremented multiple times in the past 2 releases.

[Note: In the latest PR I've added validation to ensure the state_schma_version number is correct and the testers (Who has created question suggestion for testing the contributor dashboard in past) aren't able to use the contributor-dashboard page as their old suggestions are throwing validation errors.

> Are we going to reconstitute this data? How?

This is going to be a permanent deletion and we aren't going to regenerate this data.

Reply:
Ah, that's really helpful--thanks for the context. Per my suggestion to update the documentation above, can you also note in the docstring here that this is only meant to affect the test server since it's only affecting data for an unreleased feature?

Reply:
Done!